Simulation/ThrustModel.m

```diff
@@ -18,14 +18,32 @@
   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
   
   %calculated values
-  airspeed = sqrt(StateVector(3)^2 + StateVector(4)^2);
-  RPM = Max_Volt*Kv*Throttle;%use Throttle to scale down the max RPM 
-  Thrust = 4.392399*10^(-8)*RPM*diameter^(3.5)*pitch^(-.5)*((4.23333*10^(-4))*RPM*pitch-airspeed)%calculate thrust (based on model in excel file)
+  
+  %%How to get Throttle:
+  battery_ratio = SimData.battery_ratio;
+  Raw_throttle = SimData.TestCase.GetThrottle(SimData.Time);
+  Throttle = battery_ratio * Raw_throttle; % a percent representing the power available in the battery
+  
+  
+  voltage = Max_Volt * Throttle; %The voltage on the motor (a scaled down version of the max voltage)
```

Comment:
After some further research, I realized/remembered that Max Voltage of a 3S LiPo battery is actually 12.6V not 11.1V (See other comment regarding this)